scss/titles.scss

```diff
@@ -0,0 +1,6 @@
+.title-container{
+    align-items: center;
+    display: flex;
+    font-size: $text-medium;
+    margin: 0 10%;
```

Comment:
avoid using % ... `use px or vh instead`.
% is based in the element parents size so if the parent size is defined then all children will not vary.
In this case i would use something like `margin-bottom: 100px`.